pkgs/tools/networking/ebpf-verifier/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, substituteAll
+, pkg-config
+, cmake
+, boost
+, yaml-cpp
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ebpf-verifier";
+  version = "2023-07-15";
+
+  src = fetchFromGitHub {
+    owner = "vbpf";
+    repo = "ebpf-verifier";
+    # no major release yet
+    rev = "de14d3aa3cd2845b621faf32b599766a66e158cf";
+    hash = "sha256-gnxB8ZLbTyIYpd61T57LPKFm1MHufeVEq/qN9pu2Vpk=";
+    fetchSubmodules = true;
+  };
+
+  patches = [
+    # prevent CMake from trying to get libraries on the Internet
+    (substituteAll {
+      src = ./remove-fetchcontent-usage.patch;
```

A better name for this file:

```suggestion
      # We will download them instead of cmake's fetchContent
      src = ./000-use-local-catch2-sources.patch;
```